File: app/src/main/java/com/example/yujimomoi/a1daysummerintern/classFile/Point.java
package com.example.yujimomoi.a1daysummerintern.classFile;

/**
 * Created by yuji.momoi on 2016/07/07.
 */
public class Point {
	public double x;
	public double y;

	public Point() {
		this.x = 0;
		this.y = 0;
	}

	public Point(double x, double y) {
		this.x = x;
		this.y = y;
	}

	public Point(Point point) {
		this.x = point.x;
		this.y = point.y;
	}

	public void SetPoint(double x,double y) {
		this.x = x;
		this.y = y;
	}
}

File: app/src/main/java/com/example/yujimomoi/a1daysummerintern/classFile/Point.kt
package com.example.yujimomoi.a1daysummerintern.classFile

/**
 * Created by yuji.momoi on 2016/07/07.
 */
class Point {
    var x: Int = 0
    var y: Int = 0

    constructor() {
        this.x = 0
        this.y = 0
    }

    internal constructor(x: Int, y: Int) {
        this.x = x
        this.y = y
    }

    fun SetPoint(x: Int, y: Int) {
        this.x = x
        this.y = y
    }
}
